collectors/ClusterPropertiesCollector.py

```diff
@@ -94,3 +75,12 @@ def do_metrics(self, target, gauges, infos, states):
                     labels=[self.clusters[cluster_id]['name'],
                             self.clusters[cluster_id]['parent_dc_name'].lower()],
                     value={metric_suffix: info_value})
+
+        # self.post_metrics(self.g.name)
+        # self.post_metrics(self.i.name + '_info')
```

Comment:
We can remove this now

Reply:
With this, I removed all the `CollectorUp` Code.